packages/eslint-plugin/docs/rules/switch-exhaustiveness-check.mdx

```diff
@@ -53,6 +53,27 @@ switch (value) {
 
 Since `value` is a non-union type it requires the switch case to have a default clause only with `requireDefaultForNonUnion` enabled.
 
+### `considerDefaultExhaustiveForUnions`
+
+Defaults to false. Whether a `default` clause will be considered as making a switch statement over a union type exhaustive.
+
+By default, if a `switch` statement over a union type includes a `default` case, then it's considered exhaustive.
```

Comment:
this is the opposite way (since the option defaults to false). By default, every branch will be required to be explicitly handled, and one may opt out by enabling `considerDefaultExhaustiveForUnions`

Reply:
thank you I think I missed changing it at the same time when changing the default value.